docs/client-cache.rst

```diff
@@ -0,0 +1,49 @@
+ZEO Client Cache
+----------------
```

Comment:
Please use standard adornments for new files.

Reply:
Done.